qemu/tests/balloon_check.py

```diff
@@ -355,6 +355,64 @@ def get_win_mon_free_mem(self, session):
         else:
             self.test.fail("Failed to get windows guest free memory")
 
+    def get_disk_vol(self, session):
+        """
+        Get virtio-win disk volume letter for windows guest.
+
+        :param session: VM session.
+        """
+        key = "VolumeName like 'virtio-win%'"
+        try:
+            return utils_misc.get_win_disk_vol(session,
+                                               condition=key)
+        except Exception:
+            self.test.error("Could not get virtio-win disk vol!")
+
+    def operate_balloon_service(self, session, operation):
+        """
+        Run/check/stop/install/uninstall balloon service in windows guest
+
+        :param session: shell Object
+        :param operation: operation against balloon serive, e.g. run/status/
+                          uninstall/stop
+        :return: cmd execution status and output
+        """
+        error_context.context("%s Balloon Service in guest." % operation,
```

Comment:
if operation= "status" , the log info will be "status Balloon Service in guest." ?

Reply:
Yes.
This is used for all balloon service operations